RdlMigration/ElementNameConstants.cs

```diff
@@ -85,10 +85,10 @@ public class DataSetConstants
         public class PowerBIWrapperConstants
```

Comment:
remember to revert this file before commit

Reply:
reverted